src/allmydata/test/test_util.py

```diff
@@ -588,3 +592,33 @@ def test_exceptions(self):
             )
             self.assertEqual(result.version, "Default")
             self.assertIdentical(result, rref)
+
+
+class CPUThreadPool(unittest.TestCase):
+    """Tests for cputhreadpool."""
+
+    @async_to_deferred
+    async def test_runs_in_thread(self):
+        """The given function runs in a thread."""
+        def f(*args, **kwargs):
+            time.sleep(0.1)
```

Comment:
Why sleep here?

Reply:
... I think that's left over from a different implementation approach that needed more aggressive testing.